fhir2dataset/url_builder.py

```diff
@@ -40,10 +38,11 @@ def _compute_url(self) -> str:
 
         Returns:
             str -- corresponding API request url
-        """
+        """  # noqa
         # to do verify self.fhir_api_url finish by '/'
         search_query_url = (
-            f"{self.fhir_api_url}{self._query_graph.resources_alias_info[self.main_resource_alias]['resource_type']}?"
+            f"{self.fhir_api_url}"
```

Comment:
I'm not a fan of these lines, I think you should use os.path.join or urllib.parse.urljoin to concatenate the different parts of your URL. see documentation

Reply:
that way we dont need self.fhir_api_url to end by `/`

Reply:
Great idea! I have the impression that the urllib librairie offers nice things!

Reply:
I opened an issue #23 to fix it

Reply:
next PR works for me